Sources/ProjectDescription/ExecuteAction.swift

```diff
@@ -6,9 +6,14 @@ public struct ExecutionAction: Equatable, Codable {
     public let scriptText: String
     public let target: TargetReference?
 
-    public init(title: String = "Run Script", scriptText: String, target: TargetReference? = nil) {
+    /// The path to the shell which shall execute this script.
+    /// if it is nil, "/bin/sh" will be used
```

Comment:
If it is nil it uses the Xcode default, the fact that the default is `/bin/sh` shouldn't belongs here.
Also, it fits on one line.

Same for the similar change in TuistGraph